src/foam/nanos/u2/navigation/NavigationController.js

```diff
@@ -127,19 +127,18 @@ foam.CLASS({
       // on initlayout reset context so that navigation views will be created
       // under the correct context
       this.initLayout.then(() => {
-        this.__subSubContext__ = ctrl.__subContext__;
-        this.setNavCtx_();
+        self.setPrivate_('__subContext__', ctrl.__subContext__);
```

Comment:
Why calling setPrivate_()?

Reply:
Can't set subContext directly using the =. setPrivate seems to be the only way this works

Reply:
What was wrong with setting subSubContext?